manifests/klusterlet/management/klusterlet-registration-deployment.yaml

```diff
@@ -51,7 +51,7 @@ spec:
           {{if .ExternalServerURL}}
           - "--spoke-external-server-urls={{ .ExternalServerURL }}"
           {{end}}
-          {{if eq .InstallMode "Detached"}}
+          {{if or (eq .InstallMode "Detached") (eq .InstallMode "Hosted")}}
```

Comment:
I think this `Detached` can be removed too.

Reply:
Fixed.